app/src/main/AndroidManifest.xml

```diff
@@ -13,12 +13,14 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.SummerPractice2022TvTracker"
         tools:targetApi="31">
+        <activity
+            android:name=".login.LoginActivity"
+            android:exported="false" />
         <activity
             android:name=".MainActivity"
             android:exported="false" />
         <activity
-            android:name=".login.LoginActivity"
-
+            android:name=".DetailsActivity"
             android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
```

Comment:
this activity is set as main launcher only for your local setup to test the changes, please move the Main Launcher where it was